Gruvo/Gruvo/appsettings.json

```diff
@@ -6,6 +6,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "Gruvo": "Data Source=INTEL;Initial Catalog=Gruvo;Integrated Security=True"
+    "GruvoMSSQL": "Data Source=DESKTOP-HA2KAST\\KOVTUNSQL;Initial Catalog=Gruvo;Integrated Security=True"
```

Comment:
Your PC's name. 
The solution here may be using file-db and relative path for it (like ../../db.mdf) or something else. Dev's PC name is not fine)